amd/src/learningmap.js

```diff
@@ -515,7 +573,7 @@ export const init = () => {
     function clickHandler(event) {
         event.preventDefault();
         hideContextMenu();
-        if (event.target.classList.contains('learningmap-place') && selectedElement === null) {
+        if (event.target.classList.contains('learningmap-place') && (selectedElement === null || event.type == 'touchend')) {
```

Comment:
If going for the separate handling of touch events: Could this be realized by setting `selectedElement` to `null` in case of receiving a touchend event? Would make this a bit simpler.

Reply:
Done

Reply:
That's still present?